### ISSUE TITLE
add play/pause button

### ISSUE THREAD

**Issue by hokaso** [SKbarbon/flet_ived#3]
Hello, are there any plans to add a play/pause button? If not, I will submit the PR~

such as:

![image](https://github.com/SKbarbon/flet_ived/assets/26475781/03423702-d5e8-4c8a-b1ba-9a12d9e98c63)
![image](https://github.com/SKbarbon/flet_ived/assets/26475781/642569fc-ee00-4a21-a086-211f12c884cb)

**Comment by hokaso** [comment]
This is my edited version: https://github.com/Uni-Gal/fork-flet_ived/tree/no-audio

**Comment by SKbarbon** [comment]
Hey @hokaso ,
The problem is that its uses two separated package to load the audio and to load the video. One of tha packages does not support customizing the duration of the audio. The VideoContainer actually provides a play/puase functions, but the play function make it rePlay the video. So I think such a capability we must wait for flet to support it natively.

**Comment by hokaso** [comment]
> Hey @hokaso , The problem is that its uses two separated package to load the audio and to load the video. One of tha packages does not support customizing the duration of the audio. The VideoContainer actually provides a play/puase functions, but the play function make it rePlay the video. So I think such a capability we must wait for flet to support it natively.

Indeed, although my version allows us to play the video again from the suspended time point, it removes the audio-related ability. However, it meets my current business needs. Thank you very much for providing [flet_ived](https://github.com/SKbarbon/flet_ived), which you released in the community, is very helpful~
